CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
   - The command now uses a remote file (default) if a URL is provided or a local one if a path is provided.
   - This file can be generated by using the command `generate-fixtures-file`, which takes as an argument the URL of the source queried to retieve the data dumped to the file.
   - The command `generate-fixtures-file` uses a customizable list of datasets slugs to know which datasets to query.
+  - SpatialCoverage fixes [#2683](https://github.com/opendatateam/udata/pull/2683):
+    - The model cannot have `zones` and `geom` attributes set simultaneously anymore.
```

Comment:
I add a concern here.
Are we sure that we shouldn't set a geozone + a geometry at the same time?
We can have multiple geozones so why couldn't we have a geozone and an additional arbitrary geometry (ex a city and an arbitrary area around the city)?